src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -202,18 +202,39 @@ internal SyntaxNode MethodDeclaration(IMethodSymbol method, string name, IEnumer
             throw new NotImplementedException();
         }
 
+        private protected virtual SyntaxNode OperatorDeclaration(
+            int syntaxKind,
+            bool isChecked,
+            bool isImplicitConversion,
+            IEnumerable<SyntaxNode>? parameters = null,
+            SyntaxNode? returnType = null,
+            Accessibility accessibility = Accessibility.NotApplicable,
+            DeclarationModifiers modifiers = default,
+            IEnumerable<SyntaxNode>? statements = null)
+        {
+            throw new NotImplementedException();
+        }
```

Comment:
what's this guy?

Reply:
@CyrusNajmabadi A new method that allows me to pass `isChecked`.

Reply:
that makes sense. w hy is this virtual though and not abstract?

Reply:
I incorrectly assumed the type is subclass-able when I saw this:

https://github.com/dotnet/roslyn/blob/8f96b1b0c608df74aae94823f21f5878bf0bec23/src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs#L194-L203

Given there are already non-public abstract methods, it's indeed okay to make the new methods abstract.

This is fixed in the last commit now.